### ISSUE TITLE
Add triggering the Quay builds to bioperl Travis CI

### ISSUE THREAD

**Issue by hlapp** [bioperl/bioperl-docker#1]
Right now the [after-success script](https://github.com/bioperl/bioperl-live/blob/master/travis_scripts/trigger-dockerhub.sh) only triggers the corresponding Docker Hub build.

**Comment by hlapp** [comment]
This doesn't work right now because Quay requires setting up a "Custom Git Trigger" to have a webhook endpoint, and for those the Docker file path input field erroneously doesn't show up. I am told it's being fixed.

**Comment by hlapp** [comment]
This has since been fixed by Quay, and a custom Git trigger is now in place. However, based on the [Quay documentation for custom Git triggers](https://docs.quay.io/guides/custom-trigger.html) I can't figure out how to fire the POST request.

Essentially, the issue that has me stumped is that the JSON payload _requires_ information from the Git repository (namely the value for `commit`) which contains the Dockerfile (bioperl/bioperl-docker), which is different from the Git repository (bioperl/bioperl-live) whose build success on Travis CI is to fire the POST request:

```js
{
  "commit": "1c002dd",                                   // required
  "ref": "refs/heads/master",                            // required
  "default_branch": "master",                            // required
  "commit_info": {                                       // optional
    "url": "gitsoftware.com/repository/commits/1234567", // required
    "message": "initial commit",                         // required
    "date": "timestamp",                                 // required
    "author": {                                          // optional
      "username": "user",                                // required
      "avatar_url": "gravatar.com/user.png",             // required
      "url": "gitsoftware.com/users/user"                // required
    },
    "committer": {                                       // optional
      "username": "user",                                // required
      "avatar_url": "gravatar.com/user.png",             // required
      "url": "gitsoftware.com/users/user"                // required
    }
  }
}
```

One could of course write a bash function that fetches the needed commit ID for the desired branch  using Github's API, but that would seem clunky and more effort than should be necessary. And it doesn't address the question that either the commit ID or the branch should be needed but not both. (And that's aside from the fact that there's no documentation about what `ref` versus `default_branch` is supposed to mean, and we need to provide both.)

Bottom line, it seems that the Quay.io API for custom Git triggers isn't well thought through yet.

**Comment by hlapp** [comment]
Quay.io support says that this is currently not supported through the API, but it's an existing feature request for which I have been added for notification about completion.
